### PR TITLE
pre-build all pdf icons + make .zipped files for downloading

### PR DESCRIPTION
**Pull request by htor**
...and
* removes unused color picker after discussing with ios dev. they seldom color icons on this page, and when they do they do it themselves.
* adjusts some faulty too optimized icons

solves #107

### CODE REVIEW

File: lib/nrk-ellipsis--active.svg [hunk 1]
```diff
@@ -1 +1,3 @@
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" fill="currentColor" d="M23 12c0-6.075-4.925-11-11-11S1 5.925 1 12s4.925 11 11 11 11-4.925 11-11zM6 10a2 2 0 110 4 2 2 0 010-4zm6 0a2 2 0 110 4 2 2 0 010-4zm6 0a2 2 0 110 4 2 2 0 010-4z"/></svg>
+<svg  viewBox="0 0 24 24"  xmlns="http://www.w3.org/2000/svg">
```
Comment (janerikbr) [inline lib/nrk-ellipsis--active.svg]: I was asked to have a look at this PR. Looks like there are two spaces ahead of `viewBox`.

Comment (htor) [reply lib/nrk-ellipsis--active.svg]: thanks, was that all?

Comment (janerikbr) [reply lib/nrk-ellipsis--active.svg]: I have little experience with converting svgs and have no previous knowledge of the repo. That combo makes it a bit hard for me to spot errors, I guess (if there are any) :)

Comment (htor) [reply lib/nrk-ellipsis--active.svg]: thanks anyway, lgtm..

**Pull request merged**